Fixed LowerLimbGenerationDialog

tsalemink opened the pull request.

tsalemink committed:
Updated .ui file mayaviscenewidget reference.

tsalemink committed:
Merge branch 'master' of https://github.com/tsalemink/fieldworklowerlimb2sidegenerationstep

tsalemink committed:
Python2 to Python3 changes.

tsalemink committed:
Fixed LowerLimbGenerationDialog

hsorby commented on mapclientplugins/fieldworklowerlimb2sidegenerationstep/llstep.py:
```diff
@@ -222,7 +222,7 @@ def _process(body, bodyLandmarks):
                 )
             except mocap_landmark_preprocess.InsufficientLandmarksError:
-                print('Insufficient landmarks for preprocessing {}'.format(body))
+                print(('Insufficient landmarks for preprocessing {}'.format(body)))
```
Why does this line and others like it require the extra paren?

tsalemink replied:
I tried to use a script (2to3.py) for converting Python2 code to Python3 code and it did this for some reason. I tried to change most of them back but I missed some.

tsalemink committed:
PEP8 changes.

tsalemink committed:
Updated the way plugin dependencies are listed.

tsalemink committed:
Added additional output port.

Now, this plugin also outputs a set of LowerLimb landmarks. This output is necessary for the muscle scaling step of the musculoskeletal-scaling-workflow.

tsalemink committed:
PEP8 changes.

tsalemink committed:
Remove "identifier" attribute from configuration.

Identifiers are now automatically generated by the MAP Client.

tsalemink committed:
Revert "Remove "identifier" attribute from configuration."

This reverts commit aa3b76a49e7d7702c5899ad526581c6037cb455a.
